graphh/CGH.py

```diff
@@ -48,42 +51,84 @@ class GraphHopper(object):
                                   'time': 67674},
     ...
     """
+
+
     url = "https://graphhopper.com/api/1/"
 
     def __init__(self, api_key, premium=False):
+
         self.api_key = api_key
         self.premium = premium
 
-    def _url_handle(self, api, l_parameters):
-        """This function does an url request with given parameters
+
+
+    def _url_requests(self, api, data, request="get"):
+
```

Comment:
There should be no blank line between the function definition and the docstring (probably necessary for automatic doc generation to work properly).